### PR TITLE
added Troubleshooting & FAQ to install pages, 

### PR DESCRIPTION
**Pull request by moWerk**
from TOP4 asked questions on IRC since v1 release

### CODE REVIEW

File: styles/components/install-preparation-box.less [hunk 1]
```diff
@@ -35,5 +35,5 @@
   margin-left: auto;
   margin-right: auto;
   padding-top: 15px;
-  padding-bottom: 20px;
+  padding-bottom: 15px;
```
[the remark this answers is not in the document]
Comment (eLtMosen) [reply styles/components/install-preparation-box.less]: pure cosmetic.
With 20px bottom padding, there is no instance where the instapp prep. boxes are same height.
At 15px both boxes have same height when the page is wide enough,

[the remark this answers is not in the document]
Comment (FlorentRevest) [reply styles/components/install-preparation-box.less]: Fair enough

**Comment by FlorentRevest** [comment]
As always, great job mosen :) And also thanks for having helped so many users on IRC

**Comment by eLtMosen** [comment]
My pleasure.
The tutorial has been pushed back a little but i still plan to do it over the summer and some starting point is already there. Also the mentioned low poly landscape thingy is in the works "occasionally" as i go the length of doing my own low poly art first as there is zero cc0 content available, then trying to animate it. Goosh so many polygones to draw and color by hand. lol.

**Pull request merged**